scripts/linters/pylint_extensions.py

```diff
@@ -165,6 +174,24 @@ def visit_call(self, node):
                             func_name))
                     num_positional_args_unused -= 1
 
+        for arg in keyword_args:
+            # If there is *args and **kwargs in the functin definition skip the
```

Comment:
functin -> function

Reply:
Done.

Reply:
I don't see this fixed, please do a self-review before commenting "Done"! I see this pattern in lots of your PRs!

Reply:
Look again, it is there. Please try to look carefully before leaving any comments. As we don't get a lot of time for GSoC and we can't waste our time in reviewing PRs without looking the code file.